docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -212,25 +217,150 @@ Iceberg. In addition to the :ref:`globally available <sql-globally-available>`
 and :ref:`read operation <sql-read-operations>` statements, the connector
 supports the following features:
 
-* :doc:`/sql/insert`
-* :doc:`/sql/delete`, see also :ref:`iceberg-delete`
-* :doc:`/sql/update`
-* :doc:`/sql/merge`
-* :ref:`sql-schema-table-management`, see also :ref:`iceberg-tables`
-* :ref:`sql-materialized-view-management`, see also
-  :ref:`iceberg-materialized-views`
-* :ref:`sql-view-management`
+* :ref:`sql-write-operations`:
+
+  * :ref:`sql-schema-table-management`, see also :ref:`iceberg-schema-table-management` and :ref:`iceberg-tables`
+  * :ref:`sql-data-management`, see also :ref:`iceberg-data-management`
+  * :ref:`sql-view-management`
+  * :ref:`sql-materialized-view-management`, , see also :ref:`iceberg-materialized-views`
+
+.. _iceberg-schema-table-management:
+
+Schema and table management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`sql-schema-table-management` functionality includes support for:
+
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/alter-schema`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/alter-table`
+* :doc:`/sql/comment`
+
+.. _iceberg-create-schema:
+
+CREATE SCHEMA
+~~~~~~~~~~~~~
+
+The connector supports creating schemas. You can create a schema with or without
+a specified location.
+
+You can create a schema with the :doc:`/sql/create-schema` statement and the
+``location`` schema property. The tables in this schema which have no explicit
+``location`` set in :doc:`/sql/create-table` statement, are located in a
+subdirectory under the directory corresponding to the schema location.
+
+Create a schema on S3::
+
+  CREATE SCHEMA iceberg.my_s3_schema
+  WITH (location = 's3://my-bucket/a/path/');
+
+Create a schema on a S3 compatible object storage such as MinIO::
+
+  CREATE SCHEMA iceberg.my_s3a_schema
+  WITH (location = 's3a://my-bucket/a/path/');
+
+Create a schema on HDFS::
+
+  CREATE SCHEMA iceberg.my_hdfs_schema
+  WITH (location='hdfs://hadoop-master:9000/user/hive/warehouse/a/path/');
+
+Optionally, on HDFS, the location can be omitted::
+
+  CREATE SCHEMA iceberg.my_hdfs_schema;
+
+.. _iceberg-create-table:
+
+Creating tables
+~~~~~~~~~~~~~~~
+
+The Iceberg connector supports creating tables using the :doc:`CREATE
+TABLE </sql/create-table>` syntax. Optionally specify the
+:ref:`table properties <iceberg-table-properties>` supported by this connector::
+
+    CREATE TABLE my_table (
+        c1 integer,
+        c2 date,
+        c3 double
+    )
+    WITH (
+        format = 'PARQUET',
+        partitioning = ARRAY['c1', 'c2'],
+        location = 's3://my-bucket/a/path/'
```

Comment:
A trailing slash caused a regression in the past versions. Let's avoid using it in the example.

Reply:
```
trino> CREATE SCHEMA iceberg.tiny WITH (location='s3a://tiny');
Query 20221007_094114_00002_ct5a7 failed: java.lang.IllegalArgumentException: Can not create a Path from an empty string
io.trino.spi.TrinoException: java.lang.IllegalArgumentException: Can not create a Path from an empty string
	at io.trino.plugin.hive.metastore.thrift.ThriftHiveMetastore.createDatabase(ThriftHiveMetastore.java:963)
	at io.trino.plugin.hive.metastore.thrift.BridgingHiveMetastore.createDatabase(BridgingHiveMetastore.java:160)
	at io.trino.plugin.hive.metastore.cache.CachingHiveMetastore.createDatabase(CachingHiveMetastore.java:450)
	at io.trino.plugin.iceberg.catalog.hms.TrinoHiveCatalog.createNamespace(TrinoHiveCatalog.java:184)
	at io.trino.plugin.iceberg.IcebergMetadata.createSchema(IcebergMetadata.java:609)
	at io.trino.plugin.base.classloader.ClassLoaderSafeConnectorMetadata.createSchema(ClassLoaderSafeConnectorMetadata.java:350)
	at io.trino.metadata.MetadataManager.createSchema(MetadataManager.java:592)
...
```

I've tried on Trino & Hive & MinIO this functionality without a trailing `/` and stumbled on the previously exposed issue.

Reply:
```
trino> create table iceberg.tiny.test1 (x integer) with (location='s3a://tiny/test1');
CREATE TABLE
trino> create table iceberg.tiny.test2 (x integer) with (location='s3a://tiny/test2/');
CREATE TABLE
```

On the table however, I see that creating a table is possible with and without trailing slash.
Probably the above mentioned situation is a bug.